src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpCodeActionTest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Test.Utilities.PullMemberUp;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.PullMemberUp
+{
+    public abstract class CSharpPullMemberUpCodeActionTest : PullMemberUpCodeActionTest
```

Comment:
are there multiple subclasses of this?  if not, just inline into CSharpPullMemberUpTests.